### ISSUE TITLE
After create account, message on login screen too long

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#3796]
Using maqetta-test (which I believe is the non-hosted build), after creating a new account, you go back to login screen to re-enter your credentials. The message in the gray box at this point is too wordy and wraps, and it looks funny because the error message box is only big enough for one line of text.

**Comment by jhpedemonte** [comment]
The message that says you need to verify your email before you can log in?  That comes from Orion.

**Comment by JonFerraiolo** [comment]
![Screen Shot 2013-04-09 at 3 16 05 PM](https://f.cloud.github.com/assets/706721/359527/2db5c9e2-a163-11e2-911b-744f80775cc0.png)

**Comment by jhpedemonte** [comment]
Well, I didn't have an issue with the original message.  Maybe could have just had a new line before the 2nd sentence.
